src/ui/Nav.tsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from 'react-router-dom';
+import styled from '@emotion/styled';
+import { FC } from 'react';
+
+const StyledNav = styled.nav`
+  width: 100%;
+  height: 50px;
+  background-color: var(--background);
+  color: var(--text);
+  display: flex;
+  justify-content: space-around;
+`;
+
+export const Nav: FC = () => {
+  return (
+    <StyledNav>
+      <div>
+        <Link to="/">Your Recipe</Link>
```

Comment:
Let's redirect an empty path to `/your-recipe`

And include a Link for `/your-recipe`